server/src/main/java/org/apache/iotdb/db/service/thrift/impl/DataNodeInternalRPCServiceImpl.java

```diff
@@ -345,6 +346,16 @@ public THeartbeatResp getDataNodeHeartBeat(THeartbeatReq req) throws TException
     return resp;
   }
 
+  @Override
+  public TSStatus updateRegionCache(TRegionRouteReq req) throws TException {
+    boolean result = ClusterPartitionFetcher.getInstance().updateRegionCache(req);
+    if (result) {
+      return RpcUtils.getStatus(TSStatusCode.SUCCESS_STATUS);
+    } else {
+      return RpcUtils.getStatus(TSStatusCode.CACHE_UPDATE_FAIL);
```

Comment:
The CACHE_UPDATE_FAIL code combine with the StandalonePartitionFether might bewilder for ConfigNode